app/uk/gov/hmrc/hec/services/TaxCheckService.scala

```diff
@@ -65,8 +69,11 @@ class TaxCheckServiceImpl @Inject() (
   config: Config
 )(implicit ec: ExecutionContext)
     extends TaxCheckService {
+  val key: String = "hec-tax-check"
 
-  val taxCheckCodeExpiresAfterDays: Long =
+  private val isExtractedField: String       = s"data.$key.isExtracted"
+  private val fileCorrelationIdField: String = s"data.$key.fileCorrelationId"
```

Comment:
it would be nice if this and `isExtractedField` sat in the repo itself rather than the service - the service shouldn't need to know about low level implementation details like that. It might just be that we expose two different `get` methods on the repo

Reply:
will change it back 👍